### PR TITLE
Reduce number of queries per snitch on startup by 2

### PR DESCRIPTION
**Pull request by okx-code**
It's not really "SOLID" but by using a bit of knowledge about the database structure, performance is vastly improved. I benchmarked this with 100,000 snitches and startup time is reduced from 29 seconds to <1 second.

### CODE REVIEW

File: paper/src/main/java/com/untamedears/jukealert/model/Snitch.java [hunk 1]
```diff
@@ -37,6 +37,9 @@ public class Snitch extends LocationTrackable {
 	private FieldManager fieldManager;
 	private SnitchFactoryType type;
 	private boolean active;
+	private long refreshTimer;
+
```
Comment (Protonull) [inline paper/src/main/java/com/untamedears/jukealert/model/Snitch.java:41]: Why the extra whitespace?

Comment (okx-code) [reply paper/src/main/java/com/untamedears/jukealert/model/Snitch.java:41]: no idea

**Comment by Protonull** [comment]
Seems like a good idea to get all the data out at once, and I believe your benchmarks. I'd hazard a guess that connection overhead is a massive part of the disparity? Anyway, I do think though that there needs to be some discussion this PR's flattening of appender abstraction. While they're called appenders, which calls to mind things like Log4j's appenders (which KiraBukkitGateway uses to great effect [here](https://github.com/CivMC/KiraBukkitGateway/blob/24c9843fc130b32fb94cb10a2d666e43c3e9fdcc/paper/src/main/java/com/github/maxopoly/KiraBukkitGateway/log/KiraLogAppender.java)), they would be more accurately described as modules, addons, or components since they are interfaces and stateholders unto themselves. It's an [ECS](https://en.wikipedia.org/wiki/Entity_component_system). But this PR reduces LeverToggleAppender to a more Log4j understanding, making it solely a save-to-database implementation, moving the state unto Snitch itself. This isn't necessarily a bad thing, but makes LeverToggleAppender inconsistent with the other appenders, which I think needs some discussion before it's accepted as is.

**Comment by okx-code** [comment]
What do you wish to dicuss? I thought about making the query fully dynamic to maintain the appender abstraction, where it could register a LEFT JOIN, and keep them decoupled, but I thought it wasn't worth the extra time and complexity for something that is modified relatively infrequently anyway.

**Comment by Protonull** [comment]
It's not about the query but about where the state is stored in the plugin abstraction. It should either be stored on the plugin (and thus indicate a movement towards abstraction-flattening), or stay on the appender. Your PR is weirdly doing both.

**Review by wingzero54 — APPROVED** [review]
Thank you for doing this to help startup performance

**Pull request merged**